mappings/net/minecraft/inventory/CraftingInventory.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_159 net/minecraft/inventory/CraftingInventory
+	FIELD field_1132 stacks [Lnet/minecraft/class_31;
+	FIELD field_1133 width I
+	FIELD field_1134 handler Lnet/minecraft/class_71;
```

Comment:
This name is confusing because "handler" is rather ambiguous. I suggest "screenHandler" because then you will immediately know what kind of handler it is. It does however deviate from the yarn mappings.

Reply:
I'm against this change due to it deviating from yarn - https://github.com/FabricMC/yarn/blob/bd3e21e1cd577dc1b0f4ec41cc88d8933e78c7e1/mappings/net/minecraft/inventory/CraftingInventory.mapping#L3

Moreover, you still have the field type to figure out what handler it is.